admin/resources/js/adminRoutes.ts

```diff
@@ -1,5 +1,8 @@
 import { APP_DIR } from "./adminConstants";
 
+export const homePath = (): string => `/${APP_DIR}`;
+export const homePublicPath = (): string => `/${APP_DIR}/public/${APP_DIR}`;
```

Comment:
Oh right. This was a hacky way around an annoying little issue with our single-php-container htaccess setup.

Right now, if you got to `/admin/something`, it works properly. But if you just go to the url `/admin`, you get redirected (on the server side) to `/admin/public/admin`. (Same with `/auth/public/auth`, if you try that.) 

I've spent some time before on the htaccess file and laravel setups, trying to fix it, without success. Simply redirecting from `/dir/public/dir` back to `/dir` on the _frontend_ has a hacky but working solution. 🤷 

I'd appreciate your thoughts on whether this is worth it. We could always make a separate issue highlighting the htaccess file.

Reply:
> We could always make a separate issue highlighting the htaccess file.
I think that's a good idea to make an issue. It should probably be refactored since it's not very elegant, but it does work.